### ISSUE TITLE
Add bulk deletion support to ark schedule delete

### ISSUE THREAD

**Issue by carlisia** [heptio/ark#748]
Modify ark schedule delete to use a --pattern flag, so that a user could do bulk delete with a PCRE pattern as follows ark schedule delete --pattern nginx*

This is related to this issue: https://github.com/heptio/ark/issues/295

**Comment by ncdc** [comment]
We haven't done any pattern matching anywhere else, and `kubectl`'s mechanism of selecting multiple things is to use a label selector. Do you think we need the pattern matching or is a label selector acceptable and sufficient?

**Comment by carlisia** [comment]
@ncdc agree. And also the intention for this tix is to mirror #295, which is using a selector based on a list of inputs. My bad here, will correct the description.

**Comment by carlisia** [comment]
Edited the description, please check!

**Comment by shubheksha** [comment]
I assume the fix for this would be similar to #749, so I can tackle this one too. :)

**Comment by carlisia** [comment]
@shubheksha it is! You may ping me if you can use any help with either ticket.
